### ISSUE TITLE
System Tray application's context menu 'floats'

### ISSUE THREAD

**Issue by wargus85** [linuxmint/mint20-beta#193]
Applications in the system tray - eg Nextcloud, MS Teams, Discord etc, when right clicked on to bring up the context menu, appear well above the panel. The height they appear at is aligned wit the bottom of the monitor on the right. This only happens with non-system tray icons. The volume, battery, printer and notifications context menus all appear at the correct position directly above the panel, everything else does not.

I've aligned the secondary monitor's position relative to the top and bottom of the primary monitor and notice that the system tray application context menus shift accordingly. 

![2020-06-25_08-13](https://user-images.githubusercontent.com/31986887/85639610-d3656a80-b6bb-11ea-8e50-346cfcfe76d6.png)

Where they 'float' depends on the relative location of the second monitor.

![2020-06-25_08-06](https://user-images.githubusercontent.com/31986887/85639389-2db1fb80-b6bb-11ea-8271-70d1d53fa335.png)

(Otherwise loving the new Mint :) )

**Comment by xenopeek** [comment]
I assume this is in Cinnamon and the program in the screenshot is Flameshot. I couldn't reproduce this with two different size displays. There must be something else to this to reproduce it.

**Comment by wargus85** [comment]
Correct, I am running Cinnamon. Flameshot is one example, but they're all affected. Only other thing that comes to mind might be the Nvidia Driver? I could test again with nouveau and see if that makes a difference. 

![image](https://user-images.githubusercontent.com/31986887/85661965-e3943e80-b6e9-11ea-8a6d-d1370b4a42dd.png)

EDIT: Using Nouveau made no difference.
